desktop/apps/auction/components/Layout.js

```diff
@@ -1,51 +1,58 @@
+import ArtworksByFollowedArtists from 'desktop/apps/auction/components/artwork_browser/ArtworksByFollowedArtists'
+import BuyNowSaleArtworks from 'desktop/apps/auction/components/artwork_browser/BuyNowSaleArtworks'
+import ArtworkBrowser from 'desktop/apps/auction/components/artwork_browser/ArtworkBrowser'
 import AuctionBlock from 'desktop/components/react/auction_block/auction_block'
+import AuctionInfoContainer from 'desktop/apps/auction/components/layout/auction_info'
 import Banner from 'desktop/apps/auction/components/layout/Banner'
-import ArtworkBrowser from 'desktop/apps/auction/components/artwork_browser/ArtworkBrowser'
 import Footer from 'desktop/apps/auction/components/layout/Footer'
-import AuctionInfoContainer from 'desktop/apps/auction/components/layout/auction_info'
 import MyActiveBids from 'desktop/apps/auction/components/layout/active_bids/MyActiveBids'
 import PropTypes from 'prop-types'
 import React from 'react'
-import WorksByFollowedArtists from 'desktop/apps/auction/components/artwork_browser/sidebar/WorksByFollowedArtists'
+import block from 'bem-cn-lite'
 import { connect } from 'react-redux'
 
 function Layout (props) {
   const {
     associatedSale,
     showAssociatedAuctions,
     showFilter,
-    showFollowedArtistsRail,
     showInfoWindow,
     showMyActiveBids,
     showFooter
   } = props
 
+  const b = block('auction-Layout')
+
   return (
-    <div className='auction-page'>
+    <div className={b()}>
       <Banner />
 
-      <div className='main-layout-container responsive-layout-container'>
+      <div className={b('container', 'responsive-layout-container')}>
         <AuctionInfoContainer />
 
         { showAssociatedAuctions &&
           <AuctionBlock
             sale={associatedSale}
             relatedAuction
-          /> }
+          />
+        }
 
         { showMyActiveBids &&
-          <MyActiveBids /> }
+          <MyActiveBids />
+        }
 
-        { showFollowedArtistsRail &&
-          <WorksByFollowedArtists /> }
+        <BuyNowSaleArtworks />
```

Comment:
I noticed you changed the naming below (in test) to `PromotedSaleArtworks`, can we change it here as well?